dask/dataframe/groupby.py

```diff
@@ -1012,6 +1093,32 @@ def std(self, ddof=1, split_every=None, split_out=1):
         result = map_partitions(np.sqrt, v, meta=v)
         return result
 
+    @derived_from(pd.DataFrame)
+    def cov(self, ddof=1, split_every=None, split_out=1):
+        levels = _determine_levels(self.index)
+
+        if self._slice:
+            sliced_plus = self._slice + list(self.index)
+            self.obj = self.obj[sliced_plus]
```

Comment:
@TomAugspurger this looks wrong to me -- I am trying to handle the following case:
> df.groupby(df['a'])[['b', 'c']])

aka 
> "dask/dataframe/tests/test_groupby.py::test_groupby_multilevel_getitem[cov-<lambda>3]

Reply:
Seems ok. May want to ensure `_slice` is a list before doing `+`. Not sure, but that may break with

```
.groupy('a')['b'].cov()
```

and

```
.groupby('a')[pd.Index(['a', 'b'])].cov()
```

(not sure if the second is valid).

Reply:
Is there a better way of doing this ?  the variance calculations don't

Reply:
This is also not going to support an index which is a mask.  This currently fails in the `test_dataframe_aggregations_multilevel` test with `lambda df: [df['a'] > 2, df['b'] > 1]`
> KeyError('[(True, True) (True, True)] not found in axis')]

Should `cov` handle this case ?